testing/features/support/helpers/page.rb

```diff
@@ -19,12 +19,6 @@ def save_full_page_screenshot(scenario)
       attach(path, "image/png")
     end
 
-    def save_full_page_html(scenario)
```

Comment:
If this is going, should `html_file_path` and `html_pages` from the archive list go as well?

Reply:
Yes. I'll double check that.